doc/ug/install_instructions/index.md

```diff
@@ -148,6 +148,30 @@ $ make install
 
 After installation you need to add `/tools/verilator/$VERILATOR_VERSION/bin` to your `PATH` environment variable.
 
+## Verible
+
+Verible is an open source  SystemVerilog style linter and formatting tool. The style linter is relatively mature and we use it as part of our [RTL design flow]({{< relref "doc/ug/design" >}}). The formatter is still under active development, and hence its usage is more experimental in OpenTitan.
+
+You can download and build Verible from scratch as explained on the [Verible GitHub page](https://github.com/google/verible/). But since this requires the Bazel build system the recommendation is to download and install a pre-built binary as described below.
+
+### Install Verible
+
+Go to [this page](https://github.com/google/verible/releases) and download the correct binary archive for your machine. The example below is for CentOS-8.1:
```

Comment:
All our instructions are for Ubuntu (16.04) so far, please also use Ubuntu here.

Reply:
makes sense, I'll change that.